### PR TITLE
Improve new job page title color & location label text for pt-br

### PR DESCRIPTION
**Pull request by emeraquino**
https://github.com/opensanca/opensanca_jobs/issues/73 - Switching the new job page title color to `#ff6f27`. Same orange being currently used for the brand name
https://github.com/opensanca/opensanca_jobs/issues/74 - Changing location label text (pt-br)

image

### CODE REVIEW

**Comment by emeraquino** [comment]
Some of the unit tests are relying on mock data generated by `Faker`, which sometimes happens to generate the same values for two different instances. This is causing some of the tests in the suite to fail intermittently. Check `semaphoreci` build result for this branch for reference.

Also, maybe create an issue to handle those at some point?

**Comment by duduribeiro** [comment]
Awesome @emeraquino. Yeah, create a follow-up ticket to handle this. thanks a lot

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -119,7 +119,7 @@ body {
 
 .page-title {
   text-align: center;
-  color: #fff;
+  color: #f66c35;
```
[the remark this answers is not in the document]
Comment (duduribeiro) [reply app/assets/stylesheets/application.scss]: There is one variable called `$primary-color`AFAIR. You can use it

[the remark this answers is not in the document]
Comment (duduribeiro) [reply app/assets/stylesheets/application.scss]: Yeah.. `$primary-color`

https://github.com/opensanca/opensanca_jobs/blob/master/app/assets/stylesheets/_variables.scss#L5

**Comment by emeraquino** [comment]
@duduribeiro change applied!

**Comment by duduribeiro** [comment]
Awesome @emeraquino. Thanks a lot for your contribution.

**Pull request merged**